### PR TITLE
Add play-pause button, and legend

### LINKED ISSUE #8: Insertion sort

### LINKED ISSUE #13: Create mock UI of app

### LINKED ISSUE #15: Creating bars

### LINKED ISSUE #18: Create playback button

### PR DESCRIPTION
**Pull request by seanjyjy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,4 +23,6 @@ yarn-debug.log*
 yarn-error.log*
 
 .eslintcache
-.idea/prettier.xml
\ No newline at end of file
+.idea/prettier.xml
+.idea/misc.xml
+.idea/codeStyles/Project.xml
```
Comment (keanecjy) [inline .gitignore]: I think it would be better to add the whole .idea folder? I am also going to delete the .idea folder.

**Review by keanecjy — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (seanjyjy) [reply .gitignore]: alright

Comment (keanecjy) [reply .gitignore]: Sorry was not clear earlier. I meant including the whole .idea folder in .gitignore. Maybe u could also delete the .idea from our repo?

**Review by keanecjy — APPROVED** [review]
LGTM.

**Pull request merged**